tasks/rest_state/client_rest_state.py

```diff
@@ -0,0 +1,43 @@
+from common import (cursor_visibility, render_blank_screen,
+                    set_cursor_position, wait)
+from config import CLIENT_WINDOW_HEIGHT, CLIENT_WINDOW_WIDTH
+from network import receive, send
+
+from .config_rest_state import (BLANK_SCREEN_MILLISECONDS,
+                                    CROSS_SCREEN_MILLISECONDS)
+from .utils import timer
+
+class ClientRestState:
+    def __init__(self, from_server, to_server, screen):
+        self._from_server = from_server
+        self._to_server = to_server
+        self._screen = screen
+
+    def run(self):
+        print("[STATUS] Running Rest state")
+
+        while True:
+
+            [data] = receive([self._from_server])
+            print(data)
+            if data["type"] == "request":
+                if data["request"] == "end":
+                    print('im in break')
```

Comment:
This message should be formatted as
[INFO] Break period

Reply:
I don't need these, I just used them for debugging. I'll remove these.